include/logging/log_core.h

```diff
@@ -257,6 +257,14 @@ static inline char z_log_minimal_level_to_char(int level)
 /******************************************************************************/
 /****************** Macros for standard logging *******************************/
 /******************************************************************************/
+#if defined(CONFIG_ZTEST_COVERAGE_EXCLUDE_LOG)
```

Comment:
I remember that there is a config var for enabling/disabling code coverage feature. If we can use that config var, the new  config var will unecessary. Maybe, I'm wrong.

Reply:
Got it, let me check if there is a such configuration existed, thanks for your reminder.